libs/isograph-react/src/core/read.ts

```diff
@@ -343,9 +343,12 @@ function readData<TReadFromStore>(
         } else {
           const refetchQueryIndex = field.refetchQuery;
           if (refetchQueryIndex == null) {
-            throw new Error('refetchQuery is null in RefetchField');
+            throw new Error('refetchQueryIndex is null in RefetchField');
           }
           const refetchQuery = nestedRefetchQueries[refetchQueryIndex];
+          if (refetchQuery == null) {
+            throw new Error('refetchQuery is null in RefetchField');
```

Comment:
Can you add that this is indicative of a bug in Isograph

Reply:
Done!